include/boost/decimal/detail/bit_layouts.hpp

```diff
@@ -91,7 +91,7 @@ struct IEEEl2bits
 
 struct IEEEl2bits
 {
-#if BOOST_CHARCONV_ENDIAN_LITTLE_BYTE
+#if (defined(BOOST_CHARCONV_ENDIAN_LITTLE_BYTE) && (BOOST_CHARCONV_ENDIAN_LITTLE_BYTE != 0))
```

Comment:
```suggestion
#if (defined(BOOST_DECIMAL_ENDIAN_LITTLE_BYTE) && (BOOST_DECIMAL_ENDIAN_LITTLE_BYTE != 0))
```

Reply:
In the terse form, if `OOST_CHARCONV_ENDIAN_LITTLE_BYTE` is not defined at all, then checking its value only leads to warnings (properly so) with certain warnings activated.